_includes/users-summary.html

```diff
@@ -1,18 +1,20 @@
 <div class="user-logos-container">
-{%- assign selection = site.data.users | sample: 16 -%}
+{%- assign selection = site.data.users -%}
 {%- for user in selection -%}
```

Comment:
Can we not just do `for user in site.data.users`?

Reply:
yes.. forgot about going back to that..